vg/vgimg/vgimg.go

```diff
@@ -164,6 +164,14 @@ func UseImageWithContext(img draw.Image, gc draw2d.GraphicContext) option {
 	}
 }
 
+// Image returns the image the canvas is drawing to.
+//
+// Users are not supposed to directly modify the image once the canvas
```

Comment:
OK. So then probably we should say "The dimensions of the returned image must not be modified."